Update readme and issue template. (#5773)
MinecraftForge/MinecraftForge: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Visit the forum first for issues with crashes.
title: ''
labels: Bug
assignees: ''

---

<!-- Thank you for reporting! -->

> <!-- Please go to the forum first for support with crashes and issues. -->

Reviewer: ```suggestion
<!-- For support with crashes and issues, please create a post on the forum instead: -->
```
